Aerolyzer/Aerolyzer/urls.py

```diff
@@ -20,5 +20,6 @@
 urlpatterns = [
     url(r'^app/', include('app.urls')),
     url(r'^admin/', admin.site.urls),
-	url(r'^.*', include('app.urls')),
+	#url(r'^.*', include('app.urls')),
```

Comment:
If the line code needed, don't comment it out. Instead delete it plz. Much cleaner code.